test/token/ERC20/StandardToken.test.js

```diff
@@ -417,7 +423,7 @@ contract('StandardToken', function ([_, owner, recipient, anotherAccount]) {
             await this.token.increaseApproval(spender, amount, { from: owner });
 
             const allowance = await this.token.allowance(owner, spender);
-            assert.equal(allowance, amount + 1);
+            allowance.should.be.bignumber.eq(amount + 1);
```

Comment:
Is `amount` here a BigNumber instance? We should be using BigNumber addition.

Reply:
It's not, though it probably should be. Spawned https://github.com/OpenZeppelin/openzeppelin-solidity/issues/1189